src/components/InjectedComponents/TypedMessageRenderer.tsx

```diff
@@ -43,7 +46,7 @@ export const DefaultTypedMessageTextRenderer = React.memo(function DefaultTypedM
 ) {
     return renderWithMetadata(
         props,
-        <Typography variant="body1" style={{ lineBreak: 'anywhere' }} data-testid="text_payload">
+        <Typography component="span" color="textPrimary" variant="body1" data-testid="text_payload">
```

Comment:
If you don't add lineBreak any where, it will overflow over the text in some cases.

Reply:
When did that happen?

Reply:
When text is too long

Reply:
I got this. Should control the text wrapping in parent element.